src/components/Category/Category.tsx

```diff
@@ -0,0 +1,45 @@
+import Link from "next/link";
+import { Button } from "../ui/button";
+
+const Category = () => {
+  // TO DO - get categories from database, think about categories we want
+  const categories = [
+    { name: "DESSERTS 🍰", href: "/desserts" },
```

Comment:
we want category names in url like /category/desserts/ so why rename it to [categoryId] when its not an id?